db/driver/sqlc/query.sql

```diff
@@ -0,0 +1,23 @@
+-- name: SelectUserApps :many
```

Comment:
1. Can you rename the filename from `query.sql` to something more semantic? E.g. `query_all_data.sql`
2. #PUC at the top on what / why we're using this for.

Reply:
The SQLC convention is to just have a single file called `query.sql` which contains all SQL queries used to autogenerate the Go code.In our case, there is only one query to get the User Apps.

I will update the comment to make this clear.